app/services/find/unescaped_query_string_service.rb

```diff
@@ -0,0 +1,24 @@
+module Find
+  class UnescapedQueryStringService
+    def initialize(base_path:, parameters:)
+      @base_path = base_path
+      @parameters = parameters
+    end
+
+    class << self
```

Comment:
Can we use https://github.com/DFE-Digital/teacher-training-api/blob/main/app/services/service_pattern.rb

Reply:
done